README.md

```diff
@@ -1,79 +1,60 @@
 # angular-cli-esri-map
 
-This tutorial shows you how to integrate the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript), which is an enterprise geospatial API, using [Angular CLI](https://github.com/angular/angular-cli).
+This branch contains a simple but complete application that uses the ArcGIS API for JavaScript, an enterprise geospatial API, and Angular CLI. It uses esri-loader, a small library that helps load ArcGIS JavaScript API modules (v3.x or v4.x) in non-Dojo applications.
 
-This repo demonstrates implementing a simple, extensible mapping component. Use this repo as a building block for adding in more advanced capabilities.
+To use arcgis-webpack-plugin instead of esri-loader to load the ArcGIS API for JavaScript modules, check out the `@TODO ADD BRANCH NAME` branch of this repo.
 
-## Dependencies
+If you would like to create your own Angular CLI project from scratch and incorporate these components, [create a new Angular CLI project](https://cli.angular.io/) and copy the `src/app/` directory of this repo to your new project. You will need to install `esri-loader` and `@types/arcgis-js-api` manually.
 
-This repo is only tested to work with the most current versions of the following dependencies.
+## Dependencies
 
-**Note: This repo is designed to work with Angular 6+ only.**
+**Note: This repo is designed to work with Angular 6+ and is only tested to work with the most current version of the following dependencies.**
```

Comment:
Hmmm. How about `This repo is only tested to work with the most current version of the following dependencies`?